README.md

```diff
@@ -1,69 +1,38 @@
 # Bot on the Clocktower
-Discord bot to assist with running a game of Blood on the Clocktower on Discord
 
-## Introduction: What does this bot do?
+A Discord bot to assist with running a game of Blood on the Clocktower on Discord
+
+# Introduction: What does this bot do?
 
 To easily play Blood on the Clocktower via Discord voice channels, you need:
```

Comment:
I think we can simplify this significantly, or remove it entirely. We don't really have to explain how Blood on the Clocktower works, I think.